README.md

```diff
@@ -51,7 +51,9 @@ This repository is being development under a task order of the Agile Blanket Pur
   - AWS_REGION
   - AWS_BUCKET_NAME
   - SUDS_API_URL
-    - To point at mocks, update this to be `http://localhost:${PORT}/mocks/basic`
+    - To use the moxai dependency and point at the mock API, update this to be `http://localhost:${PORT}/mocks/basic`
+    - This application uses the [moxai node package](https://www.npmjs.com/package/moxai) as a placeholder mock API. The `/mocks/basic.json` file maintains the API endpoint schema.
+    - The `controllers/index.js` file has random control number generation logic that should be removed when the mock API is replaced with the Basic API.
```

Comment:
Not sure this should be here since it isn't really specific to the env vars and since a note about it is made below.